resources/views/layouts/app.blade.php

```diff
@@ -56,6 +40,7 @@
         <script src="{{ asset('js/footer.js') }}" defer></script>
         {{-- Chart.js --}}
         <script src="https://cdn.jsdelivr.net/npm/chart.js@4.4.1/dist/chart.umd.min.js" defer></script>
```

Comment:
I installed Chart.js instead of CDN with [bc7a2e3](https://github.com/kredo-school/16th_rankshty/pull/76/commits/bc7a2e3c62c79af52a70ee55945c48e088b00451).